cleaner/muc/tests/phpunit/output/index_renderer_test.php

```diff
@@ -127,10 +127,13 @@ public function test_it_downloads_environment_config_file() {
     }
 
     private function get_page() {
+        global $CFG;
+
         ob_start();
         try {
             (new controller())->index();
             $html = ob_get_contents();
+            $html .= $CFG->closingtags ?? ''; // Fix for MDL-79276, if null then set fallback to empty string.
```

Comment:
as a fix this seems fine, but I'm unclear why test_it_downloads_environment_config_file exists and what it is asserting

So ok to land it, but if there isn't a good reason for this also happy to remove the unit test (or add some docs explaining why its useful)